build-gluster-org/jobs/rpm.yml

```diff
@@ -7,9 +7,15 @@
       - fedora:
           build_flag: fedora-30-x86_64
```

Comment:
Let's remove F30 configuration. Also, didn't we upgrade to fedora32?

Reply:
F32 didn't seems to support F30 anymore, but yeah, let's remove that.